pkg/fmap/fmap.go

```diff
@@ -86,6 +86,22 @@ type Metadata struct {
 	Start uint64
 }
 
+// HeaderValid A basic check whether the fmap header is valid
+func HeaderValid(H *Header) bool {
+	const HeaderSize = 56
+	const AreaSize = 42
```

Comment:
Instead of these two constants, can you use `binary.Size(&Header{})` and `binary.Size(&Area{})`?

Reply:
ok thx